server/bin/pbench-index.py

```diff
@@ -676,17 +246,5 @@ def sigquit_handler(*args):
         help="Perform re-indexing of previously indexed data",
     )
     parsed = parser.parse_args()
-    try:
-        # The SIGTERM handler is established around main() to make it easier
-        # to handle it cleanly once established.  We also make sure both
-        # SIGQUIT and SIGINT are ignored until we are ready to deal with them.
-        signal.signal(signal.SIGTERM, sigterm_handler)
-        signal.signal(signal.SIGINT, signal.SIG_IGN)
-        signal.signal(signal.SIGQUIT, signal.SIG_IGN)
-        status = main(parsed, run_name)
```

Comment:
Is there a reason this was moved to the new module?  Don't we still need the `SIGTERM` handling here?

Reply:
Function handling of process_tb and collect_tb takes place in main(), if the error is raised in process_tb and collect_tb it needs to be handled in main(). I didn't think we need sigtern_handler here can you please clarify.

Reply:
This is the one place where we can handle `SIGTERM`.  Not sure how the module restructuring provides for this in the new code.

Reply:
Got the point